platform/darwin/src/MGLShape.mm

```diff
@@ -94,9 +94,7 @@ - (BOOL)isEqual:(id)other
 
 - (NSUInteger)hash
 {
-    NSUInteger hash;
-    hash += _title.hash;
-    hash += _subtitle.hash;
+    NSUInteger hash = _title.hash + _subtitle.hash;
```

Comment:
A XOR over the values could improve it but I'm ok with it because this is an abstract class and can't be used directly. All concrete subclasses include the hash of the GeoJSON dictionary.